source/blog/2017-03-29-up-and-running-with-ovirt-4-1-and-gluster-storage.html.md

```diff
@@ -0,0 +1,169 @@
+
+__Storage:__ The hosted engine feature requires NFS, iSCSI, FibreChannel or Gluster storage to house the VM that will host the engine. For this walkthrough, I'm using a Gluster arbiter volume, which involves creating a replica 3 Gluster volume with two standard data bricks and a third arbiter brick that stores only file names and metadata, thereby providing an oVirt hosted engine setup with the data consistency it requires, while cutting down significantly on duplicated data and network traffic.
+
+### Installing oVirt with hosted engine
+
+I'm starting out with three test machines with 16 GB of RAM and 4 processor cores, running oVirt Node 4.1.1. I actually do the testing for this howto in VMs hosted on my "real" oVirt setup, but that "real" setup resembles what I describe below.
+
+I've identified a quartet of static IP address on my network to use for this test (three for my virt hosts, and one for the hosted engine). I've set up the DNS server in my lab to make these IPs resolve properly, but you can also edit the /etc/hosts files on your test machines for this purpose.
+
+Our host1 will need to be able to access itself and the other two nodes via passwordless ssh:
+
+```
+[host1]# ssh-keygen
+
+[host1]# ssh-copy-id root@$HOST1
+
+[host1]# ssh-copy-id root@$HOST2
+
+[host1]# ssh-copy-id root@$HOST3
+```
+
+Next, open up a web browser and visit your first host at port 9090 to access the cockpit web interface. Log in with the machine's root account, click the "Virtualization" tab at the top of the screen, and then click the "Hosted Engine" link in the left sidebar. Select the radio button next to "Hosted Engine with Gluster" and hit the "Start" button.
+
+![](uarwo41-cockpit-1.png)
+
+The dialog window that appears contains a series of steps through which you provide gdeploy with the information it needs to configure your three nodes for running ovirt with gluster storage, starting with the hosts you want to configure.
+
+![](uarwo41-gdeploy-1.png)
+
+Click next to accept the defaults in step two, and then in step three, specify the gluster volumes you want to create. The cockpit gdeploy plugin autofills some values here, including a volume for the engine, a data volume, and a second data volume called vmstore. The storage domains you'll need for a minimal ovirt install are engine, data, export and iso, and so these are the ones I create: 
+
+![](uarwo41-gdeploy-3.png)
+
+Click "Next" to hit step four, where we'll specify the brick locations for our volumes. Again, the plugin prefills some values here, which aren't likely to be correct for your environment, so pay close attention here. In my test environment, I'm using one additional disk for my gluster data, `/dev/sdb`, and I'm specifying one brick per host per volume: 
+
+![](uarwo41-gdeploy-4.png)
+
+In the final, "Review" step, I found it necessary to click "Edit" and the following operation after the `script1` step:
```

Comment:
The hyperconverged gdeploy example still has it: https://github.com/gluster/gdeploy/blob/master/examples/hc.conf#L67. I don't know why it isn't in here, but my /dev/sdb isn't found w/o disabling multipath. I'll look into filing an issue.